examples/demo-react-native/scripts/bloatBundle.mjs

```diff
@@ -0,0 +1,21 @@
+import { appendFile } from 'fs/promises';
+const bundlePath = `${process.argv[2]}`;
+
+async function bloatBundle() {
+  const garbageToAppend = `/* ${ 'junk '.repeat(50000) } */`
```

Comment:
Is it effective enough? Maybe we could generate some scalable `if () {} else {}` labyrinth to make sure that the AST parser chokes?

Reply:
Yes it is, it reproduced the error we had before [Amit's fix](https://github.com/wix/Detox/pull/3504) for the RN .68 upgrade. 
Although, adding more coverage by choking the AST won't hurt this test.

Reply:
mmm, @noomorph maybe you have an idea what kind of labyrinth would choke the AST parser? Also, how would you test that the AST is choked?